src/trackers/TOS.py

```
@@ -0,0 +1,138 @@
+            "WoLFHD",
+            "EXTREME",
+            "Slay3R",
+            "3T3AM",
+            "BARBiE",
+        ]
+        pass
+
+    async def get_category_id(
+        self,
+        meta: dict[str, Any],
+        category: str = "",
```

Ruff (0.14.11)

Unused method argument: `category`

(ARG002)